Makefile

```diff
@@ -102,11 +102,14 @@ uninstall:
 
 clean:
 	$(RM) -r out/Makefile $(NODE_EXE) $(NODE_G_EXE) out/$(BUILDTYPE)/$(NODE_EXE) \
-                out/$(BUILDTYPE)/node.exp
+	out/$(BUILDTYPE)/node.exp
```

Comment:
I think the indentation in the current `Makefile` makes more sense because this is a continuation of the previous line and not a new command.